Codeinwp/neve
# Feat: Form fields styling [closes #2033]

### Summary
- Adds non-responsive spacing control;
- Adds inline select control;
- Adds form fields styling;
- Adapts `ControlWithLink` component to allow control focus;
- Uses new `InlineSelect` component for old typeface control;


### Will affect visual aspect of the product

YES

### Test instructions


- Check out the `Customizer > Form Fields` options;
- These should be applied to all form fields;
- The `Label Spacing` option should not be applied on fields for the search forms;
- Checkout form should get the style from the options;

---

_Note: Some of the options were changed from what is in the figma project._


Closes #2033.
